evap/staff/tools.py

```diff
@@ -25,8 +27,16 @@ def forward_messages(request, success_messages, warnings):
             messages.warning(request, warning)
 
 
+class ImportType(Enum):
+    User = 'user'
+    Contributor = 'contributor'
+    Participant = 'participant'
+    Semester = 'semester'
+    UserBulkUpdate = 'user_bulk_update'
+
+
 def generate_import_filename(user_id, import_type):
-    return settings.MEDIA_ROOT + '/temp_import_files/' + str(user_id) + '.xls' + '.' + import_type
+    return settings.MEDIA_ROOT + '/temp_import_files/' + str(user_id) + '.xls' + '.' + import_type.value
```

Comment:
Wouldn’t this be a place to use `os.path.join` or a f-string? Also, it feels a bit counter-intuitive to put the import type after the `.xls` extension rather than between the filename and the “real” extension. But if this is just how conventions are, I’m fine like it is

Reply:
Good point. I think I never really spend a thought on the order here.